content_handler/vulkan_rasterizer.cc

```diff
@@ -43,7 +43,7 @@ bool WaitForFirstDisplayDriver() {
     return false;
   }
 
-  status = ioctl_vfs_watch_dir_v2(fd.get(), &wd);
+  status = ioctl_vfs_watch_dir(fd.get(), &wd);
```

Comment:
Ah, yeah. I did this somewhat mechanically looking for these. I can do another CL.